modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/servlet/JSResolveModulesServlet.java

```diff
@@ -120,9 +120,11 @@ protected void service(
 			body = URLDecoder.decode(
 				body, httpServletRequest.getCharacterEncoding());
 
-			body = body.substring(8);
+			if (!body.isEmpty()) {
```

Comment:
Wouldn't it be safer to check if `method.equals("OPTIONS")` in the `if` above? :thinking:

Reply:
But yeah, it's possible that this makes sense too, because if someone sends an empty `POST`, this would prevent its failure :+1: